version.mk

```diff
@@ -1,4 +1,5 @@
 MAJOR=2
 MINOR=8
 BUGFIX=0
+API=3
```

Comment:
@ryandoherty suggested since it replaced the old schemaver of 2, it would start with 3.